public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -195,6 +197,14 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
   public static Component = ({ model }: SceneComponentProps<RowsLayoutManager>) => {
     const { rows } = model.useState();
     const styles = useStyles2(getStyles);
+    const dashboard = getDashboardSceneFor(model);
+
+    // If we are top level layout and have no children, show empty state
+    if (model.parent === dashboard && rows.length === 0) {
+      return (
+        <DashboardEmpty dashboard={dashboard} canCreate={!!dashboard.state.meta.canEdit} key="dashboard-empty-state" />
+      );
```

Comment:
@mdvictor did you add these? where did they come from?

Reply:
@mdvictor the point of this implementation was to keep the orders empty as we need specific empty states there , if we are to duplicate the same code 3 times I would not have placed it inside each layout

Reply:
@torkelo sorry about that, I didn't read the description thoroughly. I've reverted just these changes [here](https://github.com/grafana/grafana/pull/100218)

Reply:
no worries